src/registry_schemas/schemas/notice_of_withdrawal.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/notice_of_withdrawal",
+    "type": "object",
+    "definitions": {},
+    "title": "Notice of Withdrawal Filing",
+    "required": [
+        "noticeOfWithdrawal"
+    ],
+    "properties": {
+        "noticeOfWithdrawal": {
+            "type": "object",
+            "required": [
+                "filingId"
+            ],
+            "properties": {
+                "filingId": {
+                    "type": "integer",
+                    "title": "IDs for the FED filings."
+                },
+                "identifier": { "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/business#/definitions/identifier" },
```

Comment:
I'm not sure whether we need the `identifier` here, since we always have the `business` as part of the filing header: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/example_data/schema_data.py#L33C8-L42C10
Maybe double-check it?

Reply:
Or is it possible to consider referencing the `anyOf` structure of [changeOfName](https://github.com/bcgov/business-schemas/blob/5579aaf9ad355b9638ea2d7fa5feeb692f2f1869/src/registry_schemas/schemas/change_of_name.json) to handle this situation?

Reply:
> Or is it possible to consider referencing the `anyOf` structure of [changeOfName](https://github.com/bcgov/business-schemas/blob/5579aaf9ad355b9638ea2d7fa5feeb692f2f1869/src/registry_schemas/schemas/change_of_name.json) to handle this situation?

what will be the options in the `anyOf`? 🤔 
`filingId` is required, we cannot withdraw a filing without knowing that.

Reply:
Good catch...yeah `identifier` is optional. I would like to say yes that we can remove it.
PS. the filing schema is [here](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/filing.json).